ui.tests/test-module/specs/documentOfRecordTest.spec.js

```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ * ADOBE CONFIDENTIAL
+ * ___________________
+ *
+ * Copyright 2023 Adobe Systems Incorporated
+ * All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Adobe Systems Incorporated and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Adobe Systems Incorporated and its
+ * suppliers and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Adobe Systems Incorporated.
+ */
+
+describe('Document of Record Test', () => {
+  const formId = "L2NvbnRlbnQvZm9ybXMvYWYvY29yZS1jb21wb25lbnRzLWl0L3NhbXBsZXMvZG9jdW1lbnQtb2YtcmVjb3JkL2Rvci1mb3Jt";
```

Comment:
Please get the form id from list forms API, you can check here for reference, https://github.com/adobe/aem-core-forms-components/blob/06405e500db344e58f037703f4c7cd70a1e09f31/ui.tests/test-module/libs/support/commands.js#L421. formId is an implementation detail and it should not be exposed to customer,